intermediate_source/reinforcement_q_learning.py

```diff
@@ -79,7 +79,7 @@
 import torchvision.transforms as T
 
 
-env = gym.make('CartPole-v0', new_step_api=True, render_mode='single_rgb_array').unwrapped
+env = gym.make('CartPole-v0', render_mode='rgb_array').unwrapped
```

Comment:
This will make it compatible with both v0.25(which is installed on Colab) and new gym version
```suggestion
if gym.__version__ < '0.26':
    env = gym.make('CartPole-v0', new_step_api=True, render_mode='single_rgb_array').unwrapped
else:
    env = gym.make('CartPole-v0', render_mode='rgb_array').unwrapped
```